frontend/package.json

```diff
@@ -172,61 +173,6 @@
     "last 2 versions",
     "not ie <= 8"
   ],
-  "jest": {
-    "testEnvironment": "jsdom",
-    "setupFiles": [
-      "<rootDir>/.jest/register-context.js",
-      "<rootDir>/.jest/environment.js",
-      "jest-canvas-mock"
-    ],
-    "setupFilesAfterEnv": [
-      "<rootDir>/.jest/jest-setup.js"
-    ],
-    "moduleFileExtensions": [
-      "js",
-      "jsx",
-      "json",
-      "vue"
-    ],
-    "transform": {
-      "^.+\\.vue$": "@vue/vue2-jest",
-      ".+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$": "jest-transform-stub",
-      "^.+\\.jsx?$": "babel-jest"
-    },
-    "transformIgnorePatterns": [
-      "node_modules/(?!(vuetify|vee-validate|axios))"
-    ],
-    "moduleNameMapper": {
-      "^@/(.*)$": "<rootDir>/src/$1"
-    },
-    "modulePaths": [
-      "<rootDir>/node_modules/"
-    ],
-    "roots": [
-      "<rootDir>",
-      "<<rootDir>/../../common"
-    ],
-    "snapshotSerializers": [
-      "<rootDir>/node_modules/jest-serializer-vue-tjw"
-    ],
-    "testMatch": [
-      "**/tests/unit/**/*.spec.(js|jsx|ts|tsx)",
-      "**/__tests__/*.spec.(js|jsx|ts|tsx)",
-      "**/tests/infrastructure/**/*.spec.(js|jsx|ts|tsx)"
-    ],
-    "testURL": "http://localhost/",
-    "coverageDirectory": "data/coverage",
-    "collectCoverage": true,
-    "collectCoverageFrom": [
-      "src/**/*.{js,vue}",
-      "common/**/*.js",
-      "!src/components/print/print-react/generatePdf.js"
-    ],
```

Comment:
Maybe we need to use this config also for the v8 coverage.
Should i do this in this PR, or can we do this in another PR?

Reply:
I would merge this and then see whether we have any coverage problems, and solve those later.